isubit/luggage_seo
# LUGG-1099 Add seo_checker settings for all luggage content types

To test:
- Pull down this branch on a local copy of luggage
- Run `drush fra -y; drush cc all`
- Go to `admin/config/content/seo_checker` and verify that the all of the content types are checked under the `SEO CHECKER PER CONTENT TYPE` header
